### ISSUE TITLE
Incorrect mapping for UTC on windows machines

### ISSUE THREAD

**Issue by MartinDawson** [capaj/set-tz#7]
So, basically the problem is on windows the mappings from UTC actually sets it to GMT timezone. for example:

"Mon Jan 06 2020 00:00:00 GMT+0000 (Greenwich Mean Time)"

Instead it should be this:

"Mon Jan 06 2020 00:00:00 GMT+0000 (Coordinated Universal Time)"

As it is on macbooks.

This answer has the solution: https://stackoverflow.com/questions/24394610/noda-time-iana-mapping-of-etc-utc-to-windows-timezone

> As Jon pointed out, the CLDR mapping is not there for "Etc/UTC". Instead, the CLDR maps the Windows "UTC" time zone to "Etc/GMT". IMHO - this is in error.

https://stackoverflow.com/questions/17348807/how-to-translate-between-windows-and-iana-time-zones/17348822#17348822

Basically instead of mapping to GMT it should go to UTC to provide a consistent experience with macs and ubuntu otherwise we have out of sync times between OS.
